docs/modules/usage/how-to/cli-mode.md

```diff
@@ -41,7 +41,7 @@ LLM_MODEL="anthropic/claude-3-5-sonnet-20240620"
 3. Set `LLM_API_KEY` to your API key:
 
 ```bash
-LLM_API_KEY="abcde"
+LLM_API_KEY="sk_test_4eC39HqLyjWDarjtT1zdp7dc"
```

Comment:
I assume this key isn't real, but why such a real-ish looking key? If you want a better pattern than "abcde", then maybe "sk_test_12345" will suffice?

Reply:
Will do

Reply:
Updated